src/backend/aspen/workflows/nextstrain_run/build_config.py

```diff
@@ -0,0 +1,86 @@
+from math import ceil
+
+from aspen.database.models import TreeType
+from aspen.workflows.nextstrain_run.builder_base import BaseNextstrainConfigBuilder
+
+
+def builder_factory(tree_type: TreeType, group, template_args, **kwargs):
+    # This is basically a router -- We'll switch between which build types
+    # based on a variety of input.
+
+    # TODO we probably don't want to specify a literal filename in our workflow run!
+    # This is just a bandaid for now until we fix the workflow input side of tree builds.
+    mega_template = "/usr/src/app/aspen/workflows/nextstrain_run/builds_templates/mega_template.yaml"
+
+    if tree_type == TreeType.TARGETED:
+        return TargetedBuilder(mega_template, group, template_args, **kwargs)
+    if tree_type == TreeType.OVERVIEW:
+        return OverviewBuilder(mega_template, group, template_args, **kwargs)
+    if tree_type == TreeType.NON_CONTEXTUALIZED:
+        return NonContextualizedBuilder(mega_template, group, template_args, **kwargs)
+    raise Exception("Unknown build type")
+
+
+class OverviewBuilder(BaseNextstrainConfigBuilder):
+    subsampling_scheme = "OVERVIEW"
+    crowding_penalty = 0.1
+
+    def update_subsampling(self, config, subsampling):
+        if self.group.name == "Chicago Department of Public Health":
+            subsampling["group"]["query"] = "--query \"((location == '{location}') & (division == '{division}')) | submitting_lab == 'RIPHL at Rush University Medical Center'\""
```

Comment:
@danrlu There are probably multiple ways to accomplish this, but this seems simplest for right now. Is this correct?